FinniversKit/Sources/Components/ContractActionView/ContractActionView.swift

```diff
@@ -82,3 +133,9 @@ public class ContractActionView: UIView {
         delegate?.contractActionView(self, didSelectActionButtonWithUrl: buttonUrl)
     }
 }
+
+private extension UIColor {
+    class var backgroundCircleColor: UIColor {
+        UIColor(red: 0.89, green: 0.89, blue: 0.91, alpha: 1.00)
```

Comment:
Finnes det ikke egen fargekode for denne?

Reply:
Denne skulle ikke med, noen rester igjen fra testing. Har fjernet den nå.